modules/auxiliary/scanner/ipmi/ipmi_dumphashes.rb

```diff
@@ -265,6 +244,51 @@ def write_output_files(rhost, username, sha1_salt, sha1_hash)
     end
   end
 
+  def service_data
+    {
+      address: rhost,
+      port: rport,
+      service_name: 'ipmi',
+      protocol: 'udp',
+      workspace_id: myworkspace_id
+    }
+  end
+
+  def report_hash(user, hash)
+    credential_data = {
+      module_fullname: self.fullname,
+      origin_type: :service,
+      private_data: hash,
+      private_type: :nonreplayable_hash,
+      jtr_format: 'rakp',
+      username: user,
+    }.merge(service_data)
+
+    login_data = {
+      core: create_credential(credential_data),
+      status: Metasploit::Model::Login::Status::UNTRIED
+    }.merge(service_data)
+
+    create_credential_login(login_data)
+  end
+
+  def report_cracked_cred(user, password)
+    credential_data = {
+      module_fullname: self.fullname,
+      origin_type: :service,
+      private_data: password,
+      private_type: :password,
+      username: user,
+    }.merge(service_data)
+
+    login_data = {
+      core: create_credential(credential_data),
+      status: Metasploit::Model::Login::Status::UNTRIED
+    }.merge(service_data)
+
+    create_credential_login(login_data)
```

Comment:
Hey @jvazquez-r7, there's' a create_cracked_credential method. Do you need to use that instead, or we're good with create_credential_login?

https://github.com/rapid7/metasploit-credential/blob/6888dd5115f63feb93935a3665eb8fff2b18e17d/lib/metasploit/credential/creation.rb#L21